mmf/modules/embeddings.py

```diff
@@ -410,9 +408,9 @@ def forward(
                 )
             else:
                 position_ids_visual = torch.zeros(
-                    *visual_embeddings.size()[:-1],
```

Comment:
For my information, does `*` causes issues in torch scripting?

Reply:
Static size inference is not yet possible with torchscript.